libs/shapes/DeformedLayeredCircle.cpp

```diff
@@ -0,0 +1,60 @@
+#include "DeformedLayeredCircle.h"
+#include "soo_coordinates.h"
+
+namespace soo {
+
+DeformedLayeredCircle::DeformedLayeredCircle(const ofVec2f& center,
+                                             const unsigned long num_layers,
+                                             const float spacing_between_layers,
+                                             std::vector<float>& radii)
+    : center_(center) {
+  layers_.reserve(num_layers);
+
+  for (unsigned long i{0}; i < num_layers; i++) {
+    ofPath layer = this->GenerateLayer(radii);
+    layers_.push_back(layer);
+
+    bool next_layer = this->DecreaseRadii(radii, spacing_between_layers);
+    if (!next_layer) {
+      break;
+    }
```

Comment:
This could be rewritten in 2 lines:
```cpp
layers_.push_back(this->GenerateLayer(radii));
this->DecreaseRadii(radii, spacing_between_layers) ? continue : break;
```